packages/component-base/src/virtualizer-iron-list-adapter.js

```diff
@@ -704,6 +704,20 @@ export class IronListAdapter {
     }
   }
 
+  /**
+   * Sets the scroll height, that's the height of the content
+   * @override
+   */
+  _updateScrollerSize(forceUpdate) {
+    super._updateScrollerSize(forceUpdate);
```

Comment:
Can we avoid the extra call if we move this to an `else`-block?

Reply:
Any call to `super._updateScrollerSize` updates the estimated scroller size. At that point, the if block would not matter since the `_estScrollHeight` will have the older estimation.